libs/judicial-system/types/src/lib/dateLog.ts

```diff
@@ -0,0 +1,26 @@
+export enum DateType {
+  COURT_DATE = 'COURT_DATE',
+}
+
+export const getLatestDateTypeByCaseId = (
```

Comment:
Couldn't you leave out the caseId from this logic as the dateLogs returned are already filtered out to be relevant to the current case? Are there any situations where we'd be sending in a list of dates that include other case ids?

Reply:
Yes! Good catch. I've fixed that, lets discuss tomorrow about resolving this in the API instead of this implementation..